libcudacxx/include/cuda/std/detail/libcxx/include/__cuda/ptx/ptx_helper_functions.h

```diff
@@ -0,0 +1,47 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Part of libcu++, the C++ Standard Library for your entire system,
+// under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef PTX_HELPER_FUNCTIONS_H_
+#define PTX_HELPER_FUNCTIONS_H_
+
+_LIBCUDACXX_BEGIN_NAMESPACE_CUDA_PTX
+
+// Private helper functions
```

Comment:
The comment is not really adding something
```suggestion
```

Reply:
Removed.